src/arrayMethodReduce.js

```diff
@@ -4,8 +4,37 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, initialValue = '') {
```

Comment:
but the initial value, if not passed, should not be an empty string. it should be the first element of the array